yarn.lock

```diff
@@ -11814,6 +11819,19 @@ vue-template-es2015-compiler@^1.6.0, vue-template-es2015-compiler@^1.9.0:
   resolved "https://registry.yarnpkg.com/vue-template-es2015-compiler/-/vue-template-es2015-compiler-1.9.1.tgz#1ee3bc9a16ecbf5118be334bb15f9c46f82f5825"
   integrity sha512-4gDntzrifFnCEvyoO8PqyJDmguXgVPxKiIxrBKjIowvL9l+N66196+72XVYR8BBf1Uv1Fgt3bGevJ+sEmxfZzw==
 
+vue-typeahead-bootstrap@^2.5.1:
+  version "2.5.1"
+  resolved "https://registry.yarnpkg.com/vue-typeahead-bootstrap/-/vue-typeahead-bootstrap-2.5.1.tgz#b0adfd082148b0687a0b930aea94fe4f9df38dd6"
+  integrity sha512-KVWFDih0A+qhZTW4BVgsCKj1UYDjfKufRPTQQO4vjpPnymGAtYXBeP7Yx53nOvZumDPpU1zoGUZlAPa4gBFVjg==
+  dependencies:
+    resize-observer-polyfill "^1.5.0"
+    vue "^2.5.17"
+
+vue@^2.5.17:
+  version "2.6.12"
+  resolved "https://registry.yarnpkg.com/vue/-/vue-2.6.12.tgz#f5ebd4fa6bd2869403e29a896aed4904456c9123"
+  integrity sha512-uhmLFETqPPNyuLLbsKz6ioJ4q7AZHzD8ZVFNATNyICSZouqP2Sz0rotWQC8UNBF6VGSCs5abnKJoStA6JbCbfg==
```

Comment:
It's a bit weird that we're suddenly including another version of Vue in our lock file (we also have 2.6.11 just below this one), but I'm assuming that's just how yarn works 🤷‍♂️

Reply:
It's because it's defined as a dependency in vue-typeahead-bootstrap. 

A peerDependency would be better, but we don't have any control over it

Reply:
Yeah, I figured it was not something we had control over. Thanks for the explanation though!